### PR TITLE
Logic to handle existing user when creating account

### PR DESCRIPTION
**Pull request by SamTremko**

### CODE REVIEW

**Comment by SamTremko** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #724** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/724?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #730** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/730?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/vexl-it/vexl/730?utm_source=stack-comment).

File: apps/mobile/src/components/LoginFlow/components/SuccessLoginScreen/atoms.ts [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+      TE.Do,
+      TE.chainFirstW(() =>
+        set(createUserAtContactMsActionAtom, {
+          request: {firebaseToken: null},
+          credentials: session.sessionCredentials,
+        })
+      ),
+      TE.match(
+        (e) => {
+          safeNavigateBackOutsideReact()
```
Comment (kaladivo) [inline apps/mobile/src/components/LoginFlow/components/SuccessLoginScreen/atoms.ts]: Report error & reset to login flow

File: apps/mobile/src/components/LoginFlow/components/SuccessLoginScreen/atoms.ts [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+
+        return contactApi.checkUserExists()
+      }),
+      TE.match(
+        (e) => {
+          reportError(
+            'error',
+            new Error('Error when creating user session on login'),
+            {e}
+          )
```
Comment (kaladivo) [inline apps/mobile/src/components/LoginFlow/components/SuccessLoginScreen/atoms.ts:340]: Reset login flow

**Review by kaladivo — APPROVED** [review]
handle errors UX

**Comment by SamTremko** [comment]
### Merge activity

* **Jan 25, 11:55 AM**: [Graphite](https://app.graphite.dev/github/pr/vexl-it/vexl/730) rebased this pull request as part of a merge.
* **Jan 25, 12:00 PM**: @SamTremko merged this pull request with [Graphite](https://app.graphite.dev/github/pr/vexl-it/vexl/730).

**Pull request merged**